diesel/src/query_source/mod.rs

```diff
@@ -23,8 +23,12 @@ pub trait Queryable<ST, DB>
 where
     DB: Backend + HasSqlType<ST>,
 {
+    /// The Rust type you'd like to map from.
```

Comment:
Those changes are also part of the document `Queryable` pull request.

Reply:
I probably copied them over before I just decided to `#[allow(missing_docs)]`